lib/standards/html-elms.js

```diff
@@ -811,10 +811,9 @@ const htmlElms = {
 		allowedRoles: true
 	},
 	summary: {
-		contentTypes: ['phrasing', 'flow'],
```

Comment:
Why this change?

Reply:
like `kdb` misspelling, it's a correct to the file as summary is not phrasing or flow content.